Configuration/scripts/makeEfficiencyPlots.py

```diff
@@ -375,6 +388,10 @@ def MakeOneHist(dirName, histogramName):
     if arguments.plot_hist:
         plotting_options = "HIST"
 
+    h = TH2F("h1","",NBins[0],MinXValues[0],MaxXValues[0],110,0.,1.1)
```

Comment:
There's an interaction here with `finalMax` ~50 lines above, where the y-max will be 1.1 even if `setYMax` is called.

On line 340: `finalMax = 1.1` (then 341-342 can override)
On line 391 `1.1` --> `finalMax`

Reply:
done, good catch